examples/sdk/reactNative/android/build.gradle

```diff
@@ -17,5 +14,8 @@ buildscript {
     dependencies {
         classpath("com.android.tools.build:gradle")
         classpath("com.facebook.react:react-native-gradle-plugin")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin")
```

Comment:
do we need kotlin? all our projects are in java. Its even easier to adapt some patterns later from android. I strongly recommend keeping everything in java rather than in kotlin

Reply:
restored Java.